allure-android/src/main/kotlin/ru/tinkoff/allure/android/AllureAndroidRunner.kt

```diff
@@ -12,7 +13,14 @@ import android.support.test.runner.AndroidJUnitRunner
 @Keep
 open class AllureAndroidRunner : AndroidJUnitRunner() {
     override fun onCreate(arguments: Bundle) {
-        arguments.putCharSequence("listener", AllureAndroidListener::class.java.name)
+        val oldListeners: String? = arguments.getString("listener")
```

Comment:
usage of `getString` is not so universal as `getCharSequence`

Reply:
if you look at source code in AOSP, they used `bundle.getString(ARGUMENT_LISTENER)`